app/src/main/java/com/kickstarter/viewmodels/projectpage/ProjectAIViewModel.kt

```diff
@@ -0,0 +1,71 @@
+package com.kickstarter.viewmodels.projectpage
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.kickstarter.libs.Environment
+import com.kickstarter.models.AiDisclosure
+import com.kickstarter.ui.data.ProjectData
+import io.reactivex.Observable
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.subjects.BehaviorSubject
+
+interface ProjectAIViewModel {
+
+    interface Inputs {
+        /** Configure with current [ProjectData]. */
+        fun configureWith(projectData: ProjectData)
+        fun learnAboutAIPolicyOnKickstarterClicked()
+    }
+
+    interface Outputs {
+
+        /** Emits the current [AiDisclosure]. */
+        fun projectAiDisclosure(): Observable<AiDisclosure>
+
+        fun openLearnAboutAIPolicyOnKickstarter(): Observable<String>
+    }
```

Comment:
It may not be necessary with this as its a fairly simple view, but should we change this to not use inputs/outputs to fit with the new view standards we want to go with?

Reply:
👍  this ViewModel and Fragment are currently a placeholder, on next story MBL-901 the work for the screen itself will be done